backend/tomato/elements/generic.py

```diff
@@ -297,13 +338,62 @@ def _remove(self):
 			self.element = None
 			self.save()
 
+	def checkMigrate(self):
+		return self.state in ST_PREPARED
+	
+	def action_migrate(self,host):
```

Comment:
You don't need ``host`` here at all, interfaces cannot be on a different host.
Also this action does not make sense at all as an external API call.